### PR TITLE
Rename `Camunda Platform` to `Camunda`

### LINKED ISSUE #139: Update product name in human-readable reports
Camunda Platform is Camunda now.

https://github.com/search?q=repo%3Acamunda%2Fbpmnlint-plugin-camunda-compat%20platform&type=code

### PR DESCRIPTION
**Pull request by philippfromme**
Closes #139

### CODE REVIEW

File: test/camunda-platform/integration/history-time-to-live-errors.bpmn [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_19bxjpz" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.8.0-rc.0" modeler:executionPlatform="Camunda Platform" modeler:executionPlatformVersion="7.19.0">
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_19bxjpz" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.8.0-rc.0" modeler:executionPlatform="Camunda" modeler:executionPlatformVersion="7.19.0">
```
Comment (barmac) [inline test/camunda-platform/integration/history-time-to-live-errors.bpmn:2]: I don't think we want to change the XML properties.

**Review by barmac — CHANGES_REQUESTED** [review]
Let's keep the XML as is.

File: test/camunda-platform/integration/history-time-to-live-errors.bpmn (hunk 1 above)
Comment (philippfromme) [reply test/camunda-platform/integration/history-time-to-live-errors.bpmn:2]: Whoopsie.

Comment (philippfromme) [reply test/camunda-platform/integration/history-time-to-live-errors.bpmn]: Fixed.

**Review by barmac — APPROVED** [review]

**Pull request merged**